content/guides/advanced/build-a-complete-crowdfund-dapp.md

```diff
@@ -0,0 +1,635 @@
+---
+date: Feb 29, 2024
+difficulty: intermediate
+title: "Build a complete crowdfunding DApp on Solana with Anchor and React"
+description:
+  "Learn how to build a complete crowdfunding DApp on Solana using Anchor and
+  React. This tutorial will guide you through the process of creating a smart
```

Comment:
Please replace all instances of "smart contract" with "onchain program". We're eventually going to get this into a CONTRIBUTING guide for this repo, https://github.com/solana-foundation/developer-content/pull/201

Reply:
Can you also run Grammarly over this PR too? This will save the reviewer and you a bunch of time.